File: cpp/grammer/09_for_loop.cpp
#include <iostream>

int main() {
    int total = 0;
    for (int i = 1; i <= 5; ++i) {
        total += i;
        std::cout << "i=" << i << ", 현재 누적합=" << total << "\n";
    }
    std::cout << "최종 합계: " << total << "\n";
}

File: python/grammer/09_for_loop.py
"""9) 반복문(for): range와 누적합 패턴."""

total = 0
for i in range(1, 6):  # 1~5
    total += i
    print(f"i={i}, 현재 누적합={total}")

print("최종 합계:", total)
